spec/factories/dishes.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :dish do
+    name "MyString"
+    description "MyText"
+    price 1
+    category "MyString"
```

Comment:
You need to use { } brackets for the latest release of factory bot
eg `name {"MyString"}`

Reply:
done